app/helpers/file-system.js

```diff
@@ -89,6 +89,10 @@ module.exports = {
         cb(false);
       }
     });
+  },
+
+  appendToLog: function(datetime, message) {
+    fs.appendFileSync(config.logFilePath, datetime + " - " + message + "\n");
```

Comment:
@rodrigopavezi Is the synchronous instead of async version of this function being used on purpose?